README.md

```diff
@@ -1,48 +1,65 @@
-# Node Snapshots
+# 📸 Node Snapshots
 
-A basic utility to restore a blockchain client from a snapshot stored on AWS S3. The advantage of using a snapshot vs trying to run a sync from the genesis block of the blockchain is that the snapshot runs in hours, whereas the sync from genesis typically runs in days or weeks.
+A basic utility to restore a blockchain client from a snapshot stored on AWS S3.
 
-The snapshots can be used with the provided utility or downloaded and used independently. The list of snapshots is currently as follows (with more to follow):
+Syncing nodes from genesis typically takes days or weeks. Snapshots allow you to bootstrap nodes within a few hours.
 
-| Chain    | Client | Node Type | Snapshot Name  |
-| -------- | ------ | --------- | -------------- |
-| Ethereum | Erigon | Archive   | erigon_archive |
-| Ethereum | Geth   | Full      | geth_full      |
+## Instructions
+
+#### Requirements
+
+- Machine running a Debian-based OS.
 
-## Where Do Snapshots Live?
+#### Quick Start
 
-By default, snapshots are written to `s3://satsuma-snapshots/`. This bucket is in the us-east-1 region of AWS and it is configured for "requester pays" transfers.
+Run the following on your machine:
 
-The structure of this bucket is to prefix a key with the chain being snapshotted (eth, polygon, etc), then the name of the client, the version, and finally the snapshots themselves. Example:
+```bash
+./setup.sh
+./launcher.sh --client geth_full --datadir /data/mainnet
```

Comment:
I think this needs a bit more context to make it clear that geth_full is one of the supported options and that /data/mainnet is a user-created location